scripts/fetchai_netutils/fetch/cluster/instance.py

```diff
@@ -1,8 +1,7 @@
-import os
-import subprocess
 import fnmatch
 import json
-
+import os
```

Comment:
Out of interest there are a number of updates on these javascript files is this because of an updated set of formatting rules or something else (pycharm) ?

Reply:
Automated changes by pycharm, nothing to worry about.